MetaMask/metamask-extension
# Add categories to each changelog release

Each changelog release now has category headers. The standard ["keep a changelog"](https://keepachangelog.com/en/1.0.0/) categories are used, along with the addition of "Uncategorized" for any changes that have not yet been categorized.

The changelog script has been updated to add this "Uncategorized" header if it isn't already present, and to place any new commits under this header.

The changelog has been updated to property categorize each change in recent releases, and to place changes in older releases under the header "Uncategorized".

Relates to #10752

Manual testing steps:  

For development changelog updates:
- Run `yarn update-changelog`
- See that it places changelog entries under 'Unreleased`, under the "Uncategorized" header

For RC changelog updates with existing release header:
- Bump the `version` in `app/manifest/_base.json`, to simulate an RC-like environment
- Add a new release header to the changelog for the new version
- Run `yarn update-changelog`
- See that it places changelog entries under the new release header under the "Uncategorized" category
 
For updates without a release header:
- Bump the `version` in `app/manifest/_base.json`, to simulate an RC-like environment
- Run `yarn update-changelog`
- See that it places changelog entries under the newly added release header under the "Uncategorized" category
